.gitignore

```diff
@@ -1,34 +1,37 @@
-# Copyright 2017 Google LLC All Rights Reserved.
```

Comment:
I think this file showed up just because you changed line endings from \n to \r\n, from Linux style to Win.